### DISCUSSION TITLE
Keep it up!

### DISCUSSION THREAD

**Discussion by AgathaJarynski** [mdbootstrap/TW-Elements#1606] [Kind words & general feedback]
I've been watching the project closely for the last couple of weeks.

I think I'll wait until it comes out of beta before I start to use it in production, but I've played around with it already and the latest beta2 looks very promising - especially the optimization features by @juujisai 

Btw. I love that you credit the contributors in the changelog - more projects should do that.

**Comment by filipkappa** [comment]
Thanks! We'll keep it up for sure ;)
